### PR TITLE
fx: Styling issues

### PR DESCRIPTION
**Pull request by brett-onions**
Resolving the following tickets:

86c0fe6na: Navbar: remove padding around
86c08q705: Header is bigger than the other pages and logo is off center
86c0fe27p: Logo: image used is cropped to tightly

### CODE REVIEW

**Comment by rcrichton** [comment]
Task linked: [CU-86c08q705 Header is bigger than the other pages and logo is off center](https://app.clickup.com/t/86c08q705)

File: packages/header-app/src/header-styles.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+html {
+    margin: 0 !important;
+    padding: 0 !important;
+}
+
+body {
+    margin: 0 !important;
+}
```
Comment (drizzentic) [inline packages/header-app/src/header-styles.css:8]: Do we still need this file?

File: packages/header-app/src/root.component.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import Box from '@mui/material/Box'
 import {ThemeProvider} from '@emotion/react'
 import OpenhimAppBar from './components/openhim.appbar.component'
 import {ConfirmationProvider} from './contexts/confirmation.context'
+import './header-styles.css'
```
Comment (drizzentic) [inline packages/header-app/src/root.component.tsx:6]: Seems it's not being used

File: packages/header-app/src/components/openhim.appbar.component.tsx [hunk 3]
```diff
@@ -239,196 +248,81 @@ export default function OpenhimAppBar() {
         fontSize: '14px',
         backgroundColor: '#ffffff',
         color: '#000000',
-        boxShadow: 'none',
-        borderBottom: '1px solid #e0e0e0',
+        // boxShadow: 'none',
+        // borderBottom: '1px solid #e0e0e0',
```
Comment (drizzentic) [inline packages/header-app/src/components/openhim.appbar.component.tsx:252]: We should probably discard this properties

**Review by drizzentic — CHANGES_REQUESTED** [review]

**Review by drizzentic — APPROVED** [review]

**Pull request merged**